### ISSUE TITLE
Balance the item store costs

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#38]
```
Currently, all items have a fixed selling and buying price per item type that 
is defined in the item editor. These prices differ a lot, and currently feel 
like they are set rather arbitrarily.

I think we should look into getting some formula to calculate what the item 
cost should be, to make the item costs more consistent.

The parameters to weigh in when determining an item's cost are:
HP gain (potions)
Attack Cost AP modifier (or AP cost for weapons)
Attack Chance modifier
Critical Chance modifier
Critical hit multiplier
Damage potential (range)
Block chance
Damage resistance
Item type (weapon/wearable/potion/other)

```

Original issue reported on code.google.com by `oskar.wi...@gmail.com` on 29 Nov 2010 at 8:02

**Comment by GoogleCodeExporter** [comment]
```
Pardon me for being possibly incorrect, I'm not much of a developer, more of an 
avid gamer type.. anyways, it seems that at the very least it could be based of 
a system of different bases and prefixes.  I know that is a fairly cliche 
system but I personally find it fun and balanced.  I.e. 

Short swords cost 20gp, then qualities change it.  Bronze +5, Tin +7, Silver 
+10, etc etc.  And then long swords would be like 35g, and continue that sort 
of trend.


Also it could lead to more implementations of things like "Flaming silver 
longsword"

Longsword is like 35, plus 10 for silver, plus 50 or more for flaming = 100gp. 

To sell items just take off a percentagem unless you plan on putting in a skill 
like bartering or something

```

Original comment by `ds1904...@gmail.com` on 2 Dec 2010 at 7:40

**Comment by GoogleCodeExporter** [comment]
```
Sorry to double post but a good example of flaming silver longsword:

Longsword AP 5. Atk Ch 10, Dmg 1-4.

Silver, being lightweight could be like -1AP, +2-3Atk Ch, and +2-3 damage.

So silver longsword would be AP 4, 12 Percent Atk Ch, and 3-7 damage.

Flaming could just add damage, unless you add elemental type damages.

So the end result would be like 

"Flaiming silver longsword"

Ap 4, atk Ch 12, damage 5-10, or 3-7 w 2-3 flaming damage.
```

Original comment by `ds1904...@gmail.com` on 2 Dec 2010 at 7:44

**Comment by GoogleCodeExporter** [comment]
```
I like the suggestion from ds1904.ds to have such item series.
```

Original comment by `SamuelPl...@gmail.com` on 2 Dec 2010 at 9:56

**Comment by GoogleCodeExporter** [comment]
```
Silver as light and hard? not in real life :) As an "avid gamer" I refer to 
silver as specific substance (e.g. damage lycanthropes). This sword is more 
like mithril for me, but it's just my stereotype.
```

Original comment by `surrano` on 8 Dec 2010 at 12:08

**Comment by GoogleCodeExporter** [comment]
```
**** SPOILER ALERT **** some quest related info below...

back into original topic:
I like the idea that most items with same effect are of same price (e.g. gloves 
or boots or helm 6% == same price)
As for making price categories for persistent (equipment) items: how about 
something like fibonacci? e.g.
1% = 20gp
2% = 30gp
...
6% = 210gp
...

As for healing: I wondered if it was intentional:
- food is more expensive than potions
- more potent potions are more expensive per HP gained

As for fixing price purely by properties: 
- may be unbalanced in extreme cases (DND experience of DMG methods of magic 
item price calculation)
- special items need special price anyway, e.g. bonemeal potion 

```

Original comment by `surrano` on 8 Dec 2010 at 12:14

**Comment by GoogleCodeExporter** [comment]
```
Mh dont tell him that food is expensive, I got 30k gold of that. ;-)
```

Original comment by `SamuelPl...@gmail.com` on 8 Dec 2010 at 12:20

**Comment by GoogleCodeExporter** [comment]
```
Some(all?) items should have a required level, but equip/buy is possible. Quest 
rewards should bypass "too low level to buy" rule.
```

Original comment by `kims...@gmail.com` on 22 Dec 2010 at 3:38

**Comment by GoogleCodeExporter** [comment]
```
I know requiring levels would slow the game down a lot. But that's kind of the 
goal anyway ism't it? To extend the game till we reach level 50 or so. As it is 
know I save up to get the steel sword right off the bat. I like the idea as it 
applies to quests too. I've "finished" the game 3 times so far so I know what 
quests I can handle but newcomers might need some narrowing of paths. That 
would definitely make it easier to integrate the storyline better with the 
quests.
```

Original comment by `Captain8...@gmail.com` on 22 Dec 2010 at 5:31

**Comment by GoogleCodeExporter** [comment]
```
I really think we should figure out some formula to dynamically determine the 
store cost of items. That way, items won't be unbalanced with regards to their 
cost. 

The automatic price should be overridable though, for quest items that should 
be sellable for lesser value than the dynamic formula.
```

Original comment by `oskar.wi...@gmail.com` on 10 Feb 2011 at 6:49
- Added labels: **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
Original comment by `oskar.wi...@gmail.com` on 29 Jul 2011 at 6:38
- Changed state: **Started**
- Added labels: **Milestone-0.6.10**

**Comment by GoogleCodeExporter** [comment]
```
This was completed a while ago. Marking as ready.
```

Original comment by `oskar.wi...@gmail.com` on 5 Sep 2011 at 8:28
- Changed state: **Ready**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.10 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 21 Oct 2011 at 8:56
- Changed state: **Fixed**
